docs/medical-api/api-reference/document/get-document.mdx

```diff
@@ -14,6 +14,10 @@ This endpoint returns a URL which you can use to download the specified document
   The file name of the document.
 </ParamField>
 
+<ParamField query="conversionType" type="string">
+  The doc type to convert to. Either "html" or "pdf".
+</ParamField>
```

Comment:
We should probably mention that this will only work for XML/CDA files.

also hypernit: wrapping data in backticks will make it stand out more, ie: `"html"` or `"pdf"`